buffer/src/test/java/io/netty/buffer/AbstractByteBufTest.java

```diff
@@ -2268,6 +2268,7 @@ private void testBytesInArrayMultipleThreads(final boolean slice) throws Excepti
         final CyclicBarrier barrier = new CyclicBarrier(11);
         for (int i = 0; i < 10; i++) {
             new Thread(new Runnable() {
+                @SuppressWarnings("MisorderedAssertEqualsArguments")
```

Comment:
Static analysis in IntelliJ was triggered "MisorderedAssertEqualsArguments" warning on the `assertArrayEquals()` statements.
I made supression more targeted through `//noinspection`.